Dockerfile.dev

```diff
@@ -31,24 +23,23 @@ RUN \
         libxml2 \
         git \
         cmake \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+        # Needed by the stream component \
+        ffmpeg; \
+    rm -rf /var/lib/apt/lists/*
 
-WORKDIR /usr/src
+# Upgrade pip
+RUN pip3 install --upgrade pip
```

Comment:
This is not correct. Or pip version is pinned, this defeats that.

Reply:
Oh. Nice. Thanks.